websocket/inflight/websocket.sim.mts

```diff
@@ -0,0 +1,16 @@
+import { WebSocketServer } from "ws";
+
+export const _sendMessage = (connectionId: string, message: string) => {
+  let wss: WebSocketServer = global.wss;
```

Comment:
Is `global.wss` something that the `ws` package creates?

Reply:
No, `global.wss` is a variable created in the global scope that can be referenced elsewhere.

In this case, it was created during the initialization of the websocket in file `platform/sim/wb.mts`

https://github.com/marciocadev/winglibs/blob/d27705faf91fe16a90840e9eb48a774c45d6f25f/websocket/platform/sim/wb.mts#L9-L11